packages/input-option/src/InputOption/InputOption.style.ts

```diff
@@ -0,0 +1,131 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import {
+  fontFamilies,
+  spacing,
+  transitionDuration,
+  typeScales,
+} from '@leafygreen-ui/tokens';
+
+export const inputOptionStyles = css`
+  position: relative;
+  list-style: none;
+  outline: none;
+  border: unset;
+  margin: 0;
+  text-align: left;
+  text-decoration: none;
+  cursor: pointer;
+
+  font-size: ${typeScales.body1.fontSize}px;
+  line-height: ${typeScales.body1.lineHeight}px;
+  font-family: ${fontFamilies.default};
+  padding: ${spacing[2]}px ${spacing[2] + spacing[1]}px;
+
+  transition: background-color ${transitionDuration.default}ms ease-in-out;
+
+  &:focus,
+  &:focus-visible {
+    outline: none;
```

Comment:
do we want to explicitly unset this when focus-visible?

Reply:
Yes, otherwise if `as="button"` you'll get a default focus ring